src/components/util/api.js

```diff
@@ -0,0 +1,40 @@
+const errMessage = "데이터 불러오기 실패";
```

Comment:
보통 공통 변수의 경우 모두 대문자로 적는 게 일반적입니다!
```suggestion
const ERROR_MSG = "데이터 불러오기 실패";
```